cmd/hz/app/app.go

```diff
@@ -44,6 +46,23 @@ func New(c *cli.Context) error {
 	setLogVerbose(args.Verbose)
 	logs.Debugf("args: %#v\n", args)
 
+	// determine if code needs to be regenerated
```

Comment:
I think whether we can regulate user behavior that `hz new` is only used to create a project and if hz detect the project has been created, just exit with error: `the hz project has been created, please use update command to update your project`

Reply:
done